styles/_variables.scss

```diff
@@ -0,0 +1,52 @@
+$font--primary: 'Inter';
```

Comment:
**Suggestion** : Add a backup font for the font family as there are flashes of unstyled text when I resize the window occasionally

```suggestion
$font--primary: 'Inter', Helvetica, Arial;
```